src/api/spec/controllers/webui/users/tokens_controller_spec.rb

```diff
@@ -0,0 +1,169 @@
+require 'rails_helper'
+
+RSpec.describe Webui::Users::TokensController, type: :controller do
+  let(:user) { create(:confirmed_user, login: 'foo') }
+  let(:other_user) { create(:confirmed_user, login: 'bar') }
+
+  before do
+    Flipper[:trigger_workflow].enable
+  end
+
+  describe 'GET #index' do
+    context 'user is not logged in' do
+      it_behaves_like 'require logged in user' do
+        let(:method) { :get }
+        let(:action) { :index }
+      end
+    end
+
+    context 'user is logged in' do
+      before do
+        create(:service_token, user: user)
+        create(:workflow_token, user: user)
+        create(:rss_token, user: user)
+        create(:release_token, user: other_user)
+
+        login user
+        get :index
+      end
+
+      it { expect(response).to render_template('webui/users/tokens/index') }
```

Comment:
What would stop this from functioning? :)

Reply:
Lines `it { expect(response).to render_template('...') }` removed. As this was the only thing that was tested for tne `new` action, that test was also removed.